app/client/lib/ethereum/observeBlocks.js

```diff
@@ -19,49 +19,64 @@ Update wallet balances
 @method updateBalances
 */
 updateBalances = function() {
+    // UPDATE TOKEN BALANCES
```

Comment:
ah ok! I applied those changes because I was trying to figure out why the accounts balances weren't changing - which is how I found that the bug was the block issue. I was confused on why it didn't apply to accounts, now it's explained.